test/Npgsql.Specification.Tests/NpgsqlDataReaderTests.cs

```diff
@@ -0,0 +1,25 @@
+using AdoNet.Specification.Tests;
+using Xunit;
+
+namespace Npgsql.Specification.Tests
+{
+    public sealed class NpgsqlDataReaderTests : DataReaderTestBase<NpgsqlSelectValueFixture>
+    {
+        public NpgsqlDataReaderTests(NpgsqlSelectValueFixture fixture)
+            : base(fixture) {}
+
+        [Fact]  // https://github.com/mysql-net/AdoNetApiTest/pull/12
+        public override void GetName_works()
```

Comment:
This is merged and shipped in `-alpha3`, so it should be safe to delete this overridden method.

Reply:
Thanks, done.